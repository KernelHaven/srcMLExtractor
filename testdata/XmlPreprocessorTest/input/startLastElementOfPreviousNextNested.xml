<unit language="C">
	<block>
		<empty_stmt>;</empty_stmt>
		<cpp:ifdef>#<cpp:directive>ifdef</cpp:directive><name>A</name></cpp:ifdef></block>
	<block>
		<empty_stmt>;</empty_stmt>
		<cpp:endif>#<cpp:directive>endif</cpp:directive></cpp:endif></block>
</unit>
