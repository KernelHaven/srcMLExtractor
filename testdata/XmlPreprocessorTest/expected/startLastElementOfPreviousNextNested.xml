<unit language="C">
	<block>
		<empty_stmt>;</empty_stmt>
	</block><cpp:ifdef>#<cpp:directive>ifdef</cpp:directive><name>A</name>
		<block>
			<empty_stmt>;</empty_stmt>
		</block></cpp:ifdef>
</unit>
